avocado/core/test.py

```diff
@@ -741,7 +741,7 @@ def _run_avocado(self):
             raise exceptions.TestSkipError(details)
         except exceptions.TestCancel as details:
             stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
-            raise
+            raise  # pylint: disable=W0612
```

Comment:
Hi @ana, why do we still need this disabled? Since we are not using the details in this context, did you find any issue with the following?

```diff
diff --git a/avocado/core/test.py b/avocado/core/test.py
index fab83095..e68de32b 100644
--- a/avocado/core/test.py
+++ b/avocado/core/test.py
@@ -739,7 +739,7 @@ def _run_avocado(self):
             skip_test = True
             stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
             raise exceptions.TestSkipError(details)
-        except exceptions.TestCancel as details:
+        except exceptions.TestCancel:
             stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
             raise
         except:  # Old-style exceptions are not inherited from Exception()
@@ -754,7 +754,7 @@ def _run_avocado(self):
                     loop.run_until_complete(testMethod())
                 else:
                     testMethod()
-            except exceptions.TestCancel as details:
+            except exceptions.TestCancel:
                 stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
                 raise
```

Reply:
@beraldoleal indeed, I didn't realize that wen updating. It's updated now, hopefully it should be good to be merged.